python/src/aac/lang/_active_context.py

```diff
@@ -0,0 +1,136 @@
+"""The Active Context maintains the highly-contextual AaC DSL."""
+
+import copy
+
+from attr import Factory, attrib, attrs, validators
+
+from aac.parser import ParsedDefinition
+from aac.lang.definition_helpers import get_definition_by_name, get_definitions_by_root_key, search
+
+
+@attrs(slots=True, auto_attribs=True)
+class ActiveContext:
```

Comment:
Alternative naming could be like LanguageContext where ActiveContext specifically refers to the managed context instance.